Modules/AutomatedLab.md

```diff
@@ -0,0 +1,20 @@
+# Name: AutomatedLab
+
+## Description:
+AutomatedLab is a provisioning solution and framework that lets you deploy complex labs on HyperV and Azure with simple PowerShell scripts. It supports all Windows operating systems from 2008 R2 to 2016 including Nano Server and various products like AD, Exchange, PKI, IIS, etc. 
+
+| Works with | Author(s) | Project URL(s) | PowerShell Gallery | Creation Date | Tags |
+|------------|--------|-------------------|--------------------|---------------|------|
+| PS v5.0+ | Raimund Andrée (<a href="https://twitter.com/raimundandree" target="_blank">t</a>), Per Pedersen, Jan-Hendrik Peters (<a href="https://twitter.com/nyanhp">) | [Github](https://github.com/AutomatedLab/AutomatedLab) | nA | 2012 | Automation, Lab Deployment |
```

Comment:
Please fix the Jan-Hendrik Peters twitter URL
Use "N/A" instead of "nA"
Add the month of creation's date (if you know it )